Java/avg/array.java

```diff
@@ -12,4 +12,15 @@ public static void main(String[] args) {
             System.out.println(arr[i]);
         }
     }
+
+    public static void Even(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        int[] arr = new int[]{1, 2, 3, 4, 5};

```

Comment:
**Suggestion**: Avoid hardcoding the array within the method. Consider passing the array as a parameter to increase the method's flexibility and reusability. <br> **Code Suggestion**: 
 ```
 -        int[] arr = new int[]{1, 2, 3, 4, 5};
 +        // Consider passing the array as a parameter
 ```